container/openejb-core/src/main/java/org/apache/openejb/cdi/CdiEjbBean.java

```diff
@@ -250,7 +250,7 @@ protected T createEjb(final CreationalContext<T> creationalContext) {
             } else if (remote != null) {
                 instance = (T) remote.create();
             } else { // shouldn't be called for an MDB
-                throw new IllegalStateException("no interface to proxy for ejb " + beanContext.getEjbName() + ", is this is a MDB maybe you shouldn't use a scope?");
+                throw new IllegalStateException("no interface to proxy for ejb " + beanContext.getEjbName() + ", is this is a @MessageDriven bean, maybe you shouldn't use a scope?");
```

Comment:
In my case, it's not a message-driven bean, but I like the suggestion, maybe the wording should be something like:

```suggestion
                throw new IllegalStateException("No interface to proxy for ejb " + beanContext.getEjbName() + ", if this is a @MessageDriven bean, try not using a scope?");
```

Reply:
Thanks @bdemers - you are right. MDB means `@MessageDriven` bean here. I think that the proposed change is valid and clarifies the error message.